builtin/providers/aws/provider.go

```diff
@@ -543,6 +550,9 @@ func init() {
 			"use virtual hosted bucket addressing when possible\n" +
 			"(http://BUCKET.s3.amazonaws.com/KEY). Specific to the Amazon S3 service.",
 
+		"batch_security_groups": "Set this to batch refreshes EC2 security groups." +
+			" Speeds up refreshes when you have large numbers of security groups.",
```

Comment:
Can we put in wording that stipulates that this is a beta or "experimental" feature? At least that's how I see this feature. We believe it's totally fine and with no consequences, but it's an advanced feature of sorts that most situations do not warrant, yeah?